app/models/miq_event_definition.rb

```diff
@@ -123,6 +123,7 @@ def self.seed
   end
 
   def self.seed_default_events
+    event_sets = MiqEventDefinitionSet.all.group_by(&:name)
```

Comment:
is it operation on Array ?

what about to use sql ?
` MiqEventDefinitionSet.group(:name)`